src/opts/mod.rs

```diff
@@ -76,63 +73,76 @@ impl Opts {
 
     #[cfg(test)]
     pub fn parse_and_load_with_system_theme(
-        args: &Args,
+        args: Args,
         config: Config,
         theme: ResolvedTheme,
     ) -> Self {
         Self::parse_and_load_inner(args, config, theme)
     }
 
-    fn parse_and_load_inner(args: &Args, config: Config, fallback_theme: ResolvedTheme) -> Self {
-        let Config {
-            theme: config_theme,
-            scale: config_scale,
-            page_width: config_page_width,
-            lines_to_scroll: config_lines_to_scroll,
-            light_theme: config_light_theme,
-            dark_theme: config_dark_theme,
-            font_options: config_font_options,
-            keybindings:
-                config::KeybindingsSection {
-                    base: keybindings_base,
-                    extra: keybindings_extra,
-                },
-        } = config;
```

Comment:
I prefer destructuring all the values here as a way of future proofing that any additions to `Config` will be a hard compiler error here. Sure it's more code, but it also helps guide the programmer on what needs to be changed

It really should be done for `args` too. Not sure why I didn't